.github/workflows/changesets.yaml

```diff
@@ -117,6 +126,11 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
How does this part of the GH workflow work @plmercereau? I don't see any release created for the dashboard.

This seems to be a good alternative, although I haven't tested it yet: https://github.com/softprops/action-gh-release

Reply:
this step reads the changelog generated by changesets and releases a version from the given tag.
This follows the same logic as the internals of the changesets GH action

Reply:
You don't see any gh release yet because this step has never run since set - the two last releases failed because of the Next env vars issue

Reply:
From the [fork](https://github.com/plmercereau/nhost/actions/runs/3514555552/jobs/5888772224)

Reply:
Ah, OK. Sorry for the false alarm. ⏰